docs/usage.md

```diff
@@ -202,6 +203,170 @@ genome: 'GRCh37'
+[...]
+```
+
+**Feature barcode reference for antibody capture**
+
+File containing feature barcodes used for AB analysis (`--fb_reference`). Example:
+
+```csv
+id,name,read,pattern,sequence,feature_type
+CD3,CD3,R2,^NNNNNNNNNN(BC)NNNNNNNNN,CTCATTGTAACTCCT,Antibody Capture
+CD4,CD4,R2,^NNNNNNNNNN(BC)NNNNNNNNN,TGTTCCCGCTCAACT,Antibody Capture
+CD8,CD8,R2,^NNNNNNNNNN(BC)NNNNNNNNN,GCGCAACTTGATGAT,Antibody Capture
+CD11c,CD11c,R2,^NNNNNNNNNN(BC)NNNNNNNNN,TACGCCTATAACTTG,Antibody Capture
+CD14,CD14,R2,^NNNNNNNNNN(BC)NNNNNNNNN,TCTCAGACCTCCGTA,Antibody Capture
+CD16,CD16,R2,^NNNNNNNNNN(BC)NNNNNNNNN,AAGTTCACTCTTTGC,Antibody Capture
+CD19,CD19,R2,^NNNNNNNNNN(BC)NNNNNNNNN,CTGGGCAATTACTCG,Antibody Capture
+CD56,CD56,R2,^NNNNNNNNNN(BC)NNNNNNNNN,TCCTTTCCTGATAGG,Antibody Capture
+CD45,CD45,R2,^NNNNNNNNNN(BC)NNNNNNNNN,TCCCTTGCGATTTAC,Antibody Capture
+IgG1,IgG1_control_TotalSeqC,R2,^NNNNNNNNNN(BC)NNNNNNNNN,GCCGGACGACATTAA,Antibody Capture
+A0201_NLVPMVATV_CMV_TCR-1,NLVPMVATV_CMV_TCR-1,R2,^NNNNNNNNNN(BC)NNNNNNNNN,GGCCTCGGTCCTAGG,Antibody Capture
+```
+
+gex_target_panel = null
+vdj_inner_enrichment_primers = null
+gex_barcode_sample_assignment = null
+
+TODO: ends here
+
+## Running the pipeline
+
+The minimum typical command for running the pipeline is as follows:
+
+```bash
+nextflow run nf-core/scrnaseq --input ./samplesheet.csv --outdir ./results --genome GRCh38 -profile docker
+```
+
+This will launch the pipeline with the `docker` configuration profile and default `--type` and `--barcode_whitelist`. See below for more information about profiles and these options.
+
+Note that the pipeline will create the following files in your working directory:
+
+```bash
+work                # Directory containing the nextflow working files
+<OUTDIR>            # Finished results in specified location (defined with --outdir)
+.nextflow_log       # Log file from Nextflow
+# Other nextflow hidden files, eg. history of pipeline runs and old logs.
+```
+
+If you wish to repeatedly use the same parameters for multiple runs, rather than specifying each flag in the command, you can specify these in a params file.
+
+Pipeline settings can be provided in a `yaml` or `json` file via `-params-file <file>`.
+
+:::warning
+Do not use `-c <file>` to specify parameters as this will result in errors. Custom config files specified with `-c` must only be used for [tuning process resource specifications](https://nf-co.re/docs/usage/configuration#tuning-workflow-resources), other infrastructural tweaks (such as output directories), or module arguments (args).
+:::
+
+The above pipeline run specified with a params file in yaml format:
+
+```bash
+nextflow run nf-core/scrnaseq -profile docker -params-file params.yaml
+```
+
+with `params.yaml` containing:
+
+```yaml
+input: './samplesheet.csv'
+outdir: './results/'
+genome: 'GRCh37'
+<...>
+```
+
+You can also generate such `YAML`/`JSON` files via [nf-core/launch](https://nf-co.re/launch).
+
```

Comment:
@fmalmeida Any idea why this section is showing up in the diff? Did you add it? It reads like it should have been there before...

Reply:
I did not add. But I probably had moved when adding the `cellragermulti` section.
Maybe that is the case?

Reply:
Then it should be "missing" somewhere else in that file. I guess it's just a Github weirdity. All good then.